src/service/transaction.service.ts

```diff
@@ -188,37 +181,47 @@ export class TransactionService {
                         (modification.modificationType === AccountRestrictionModificationAction.Add ? 'Add' : 'Remove');
                     transactionFormatted += 'value:' + modification.value;
                 });
+        } else if (transaction instanceof MosaicGlobalRestrictionTransaction) {
+            transactionFormatted += 'MosaicGlobalRestrictionTransaction: ' +
+                'MosaicId:' + transaction.mosaicId.toHex() +
+                ' ReferenceMosaicId:' + transaction.referenceMosaicId.toHex() +
+                ' RestrictionKey:' + transaction.restrictionKey.toString() +
```

Comment:
this is going to be toHex() in v0.14.0 now. as restriction key has been changed to be in Hex format

Reply:
Let's merge it as it is, and I'll take this into consideration when applying 0.14.